src/Views/Game.hs

```diff
@@ -21,6 +23,7 @@ createGameDOM api gapi = do
 
   createGameChangeNameDOM api
   createUpdateMaxNumberPlayersDOM api gapi
+  createSetPasswordDOM api
```

Comment:
Haha! I realised this and then I realised how painful that would be to implement :+1: